doc/Development_Documentation/23_Installation_and_Upgrade/09_Upgrade_Notes/README.md

```diff
@@ -1,6 +1,7 @@
 # Upgrade Notes
 
 ## 10.5.0
+- [Navigation Builder] Calling the method Pimcore\Navigation\Builder::getNavigation() using extra arguments is deprecated and will be removed in Pimcore 11. Instead of using the extra arguments, it is recommended to call the method using the params array. eg: Instead of calling getNavigation($activeDocument = null, $navigationRootDocument = null,..) use getNavigation($params)
```

Comment:
```suggestion
- [Navigation Builder] Calling the method `Pimcore\Navigation\Builder::getNavigation()` using extra arguments is deprecated and will be removed in Pimcore 11. Instead of using the extra arguments, it is recommended to call the method using the params array. eg: Instead of calling `getNavigation($activeDocument = null, $navigationRootDocument = null,..)` use `getNavigation($params)`
```